PYME/Acquire/Hardware/Piezos/piezo_pipython_gcs.py

```diff
@@ -32,6 +32,11 @@
 logger = logging.getLogger(__name__)
 
 
+ADC_CHANNEL_FOR_TARGET = 0x06000500

```

Comment:
Are these specific to a particular piezo? If so, would it be better to have them as parameters to __init__ so that they can be set in the relevant hardware initialisation script for a particular setup?

Reply:
Fair point, David. They are potentially controller-specific. Have gone ahead and added/tested a change so they're now passed in the init. Thanks for the suggestion!